### PR TITLE
don't run installer role when verifying the installer

### PR DESCRIPTION
**Pull request by evgeni**
instead check that the foreman package is installed

### CODE REVIEW

**Review by wbclark — APPROVED** [review]
I guess it's safe to assume we won't have any change in packaging which causes the Foreman package to get installed before the installer is run. I always find that counter intuitive, coming from downstream where most packages get installed early as dependencies of the Satellite RPM.

A brief comment here explaining the logic in case we come back to it much later might be nice

**Review by ehelms — APPROVED** [review]

File: roles/installer/molecule/default/verify.yml [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-    foreman_installer_scenario: foreman
-    foreman_installer_verbose: true
-    foreman_installer_no_colors: false
-  roles:
-    - installer
   tasks:
+    - name: Foreman is installed
+      package:
+        name: foreman
+        state: present
```
Comment (ehelms) [inline roles/installer/molecule/default/verify.yml:9]: Maybe we should verify some artifact of the installer is present? Like https://github.com/theforeman/foreman-installer/blob/51d002636f178c0084483cb01f8aed7395a14900/hooks/boot/01-kafo-hook-extensions.rb#L11

Comment (ehelms) [reply roles/installer/molecule/default/verify.yml:9]: Or that we can access the web UI.

Comment (evgeni) [reply roles/installer/molecule/default/verify.yml:9]: I thought about the `.installed` file too, can add.

UI seems a bit overkill (and potentially slow in the container)

Comment (evgeni) [reply roles/installer/molecule/default/verify.yml:9]: added

**Pull request merged**